### PR TITLE
Updated templating/* articles to Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: templating/PHP.rst [hunk 1]
```diff
@@ -196,8 +173,8 @@ decorating the template. In the ``index.html.php`` template, define a
 
 .. code-block:: html+php
 
-    <!-- templates/Hello/index.html.php -->
-    <?php $view->extend('AppBundle::layout.html.php') ?>
+    <!-- templates/hello/index.html.php -->
```
Comment (yceruto) [inline templating/PHP.rst]: Note that templating php engine does not supports the paths provided by Twig. See related issue https://github.com/symfony/symfony/issues/25019, So it should be here `src/Resources/views/`

Comment (javiereguiluz) [reply templating/PHP.rst]: Thanks for reviewing and for reporting this error! I wasn't sure about this PHP part.

**Review by yceruto — APPROVED** [review]
Nice! :)

Surely unrelated but could you please add some note in _overriding_ section where we talking about the use of `@!` to override a template partially? Thanks! https://github.com/symfony/symfony-docs/issues/8440

**Comment by javiereguiluz** [comment]
@yceruto I've created #8741 to not forget about this. Thanks!

**Pull request merged**